### DISCUSSION TITLE
Add documentation steps to enable avatars

### DISCUSSION THREAD

**Discussion by eviscerat** [umputun/remark42#1626] [Q&A] [answered]
How do I enable avatars for annonymous comments? I want avatars to get generated for new comments.

![image](https://user-images.githubusercontent.com/17407021/232013116-852ece62-16de-44e9-98eb-82c2695b83a4.png)

Currently it seems I get no avatars. I've read the docs but nothing detailing how to add avatars.

This is my backend config minus url, secret and site:

```
AUTH_ANON=true
AUTH_EMAIL_ENABLE=false
EMOJI=true
AVATAR_TYPE=fs
AVATAR_RESIZE=0
IMAGE_FS_PATH=./var/pictures
AVATAR_FS_PATH=./var/avatars
```

**Comment by umputun** [comment]
There is nothing specific you should do to allow avatars for anonymous comments. I believe you have "not found" for all the avatars, not just anonymous ones. 

The reason is likely incorrect volume mapping, reverse proxy setup, or FS permissions. If you open the console (in your browser), you can see some details about failed calls, which will help to clarify what's going on. Another relevant piece of information - how exactly do you run the backend? Is it a dockerized setup? If so, show the docker-compose.yml or make sure you mapped `./var/picture` to some host directory/volume. And another part of the puzzle - is there a reverse proxy in front of this? If you use one, make sure it is properly configured

**Reply by eviscerat** [reply, marked as answer]
Avatars work with Github oath, only annonymous ones don't  work. I use the binary in an ec2 machine not the docker image.
```
AVATAR_TYPE=fs
AVATAR_RESIZE=0
IMAGE_FS_PATH=./var/pictures
AVATAR_FS_PATH=./var/avatars
```
I even checked the directory for the avatars and I could see avatars generated there.

Nevermind, after playing with the parameters and restarting it seems to be working.
